sdk/keyvault/keyvault-certificates/src/utils.ts

```diff
@@ -71,3 +72,28 @@ export function parseCertificateBytes(
     return toBase64(certificateBytes);
   }
 }
+
+export function mapPagedAsyncIterable<T, U>(
```

Comment:
Added helper to wrap paging primitives. This is needed since we let the generated code handle paging now instead of handrolling that logic (and it works, nice!) but the generated types don't match the public API (and they never did).